CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ IF(USE_LSSOL)
 ENDIF(USE_LSSOL)
 
 IF(USE_QUADPROG)
-  ADD_REQUIRED_DEPENDENCY("eigen-quadprog >= 1.0.0")
+  ADD_PROJECT_DEPENDENCY(eigen-quadprog REQUIRED)
```

Comment:
This is a breaking change for @Hugo-L3174: eigen-quadprog v1.0.0 doesn't have a CMake export. We have to update eigen-quadprog to v1.0.1 on robotpkg.